ge/org.osate.ge.ba/src/org/osate/ge/ba/util/BehaviorAnnexUtil.java

```diff
@@ -162,4 +167,65 @@ public AadlPackage getDataClassifierPackage() {
 		final AadlPackage pkg = root instanceof AadlPackage ? (AadlPackage) root : null;
 		return Optional.ofNullable(pkg);
 	}
+
+	/**
+	 * Helper to create an {@link EmbeddedTextValue} and get text for the {@link DispatchCondition} of a specified {@link BehaviorTransition}
```

Comment:
I don' think this should be inside `BehaviorAnnexUtil`. It should be it's own package level class.

Additionally, I think it would be better to make it a utility class (no instantiation) and just have the two public static methods for creating the text value or getting the text. May require adding private methods or slight refactoring to avoid some duplication. The constructor solves the problem but it makes it more awkward on the caller in my opinion